modules/rhizo_base/manifests/init.pp

```diff
@@ -257,6 +257,8 @@
   file { '/home/rhizomatica/bin/vars.sh':
       ensure  => present,
       content => template('rhizo_base/vars.sh.erb'),
+      owner   => rhizomatica,
+      group   => rhizomatica,
```

Comment:
I think this was intentional to avoid accidental editing of these scripts that should certainly be considered read only on the installed system. Maybe they could be someplace else.....